orchestrator/tasks.py

```diff
@@ -20,6 +20,8 @@
 
 
 def get_aware_datetime(date_str):
+    if not date_str:
```

Comment:
It was meant to sync with the Django timezone information. Honestly, I think it's safe to assume both times will be local so this is not needed.

Reply:
Can wee check this somehow? Just compare timestamp from job store and DB maybe?

Reply:
Okay I removed it and it seems like this is a necessary step since Django complains: RuntimeWarning: DateTimeField BaseModel.modified_date received a naive datetime (2021-04-01 17:25:13.119035) while time zone support is active.